sklearn/metrics/classification.py

```diff
@@ -186,6 +187,74 @@ def accuracy_score(y_true, y_pred, normalize=True, sample_weight=None):
     return _weighted_sum(score, sample_weight, normalize)
 
 
+def multilabel_confusion_matrix(y_true, y_pred):
+    """Compute True positive, False positive, True negative, False negative
+    for a multilabel classification problem.
+
+    Parameters
+    ----------
+    y_true : array-like or scipy sparse matrix, shape = (n_samples, n_labels)
+        Ground truth (correct) target label(s).
+
+    y_pred : array-like or scipy sparse matrix, shape = (n_samples, n_labels)
+        Estimated target label(s) as returned by a classifier.
+
+    Returns
+    -------
+    MCM : ndarray of shape = (n_labels, 4)
+        The multilabel confusion matrix.
+
+        Every row contains the 4 confusion values for that particular label.
+        The confusion values are in the following order - tp, fp, tn and fn.
+
+        ``MCM[i, :]`` returns the various confusion values for label i and
```

Comment:
I would rather say "for the i-th label". The original labels might not necessarily be contiguous integers.